grader/grader.py

```diff
@@ -329,9 +329,15 @@ def main(gradebook_input_file,
             print("")
             break
 
-        print("[i] Updating gradebook (" + gradebook_output_path + ")")
-        book.write_grades(gradebook_output_path)
-
+    print("[i] Updating gradebook (" + gradebook_output_path + ")")
```

Comment:
Can we leave these lines indented? This way if you need to partially stop the script it can resume where it left off.